core/new-gui/src/app/workspace/component/operator-panel/operator-label/operator-label.component.ts

```diff
@@ -1,6 +1,7 @@
 import { DragDropService } from './../../../service/drag-drop/drag-drop.service';
-import { Component, Input, AfterViewInit } from '@angular/core';
+import { Component, Input, AfterViewInit, ViewChild, ElementRef } from '@angular/core';
 import { v4 as uuid } from 'uuid';
+import { Observable, fromEvent, interval} from 'rxjs';
```

Comment:
import the Rxjs methods according to the previous APIs:

`import { Observable } from 'rxjs/Observable';
import '../../../../common/rxjs-operators';`

Also, since you are using `interval`, you might want to add `interval` to the list of imported Rxjs methods in `common/rxjs-operators`.

For instance:
add one line: `import 'rxjs/add/observable/interval';`